### ISSUE TITLE
Write alternate text on icon

### ISSUE THREAD

**Issue by koirand** [koirand/pulp#24]
For accessibility

**Comment by koirand** [comment]
Icon doesn't have alternate element, then add aria-label to "a" tag.
